ts/data/sample.ts

```diff
@@ -463,15 +463,42 @@ function getSampleMetrics(
         displayValue == null
           ? null
           : parseFloat(displayValue.replaceAll(",", ""));
-      return {
+      const misoMetric: MisoRunLibraryMetric = {
         title: metric.name,
         threshold_type: metric.thresholdType.toLowerCase(),
         threshold: getSingleThreshold(metric),
         value: revisedValue,
       };
+      if (metric.thresholdType == "BETWEEN") {
+        if (metric.maximum === undefined) {
+          throw new Error("Metric is missing maximum value");
```

Comment:
why do we only test for maximum here?

Reply:
The minimum is checked when populating it in `threshold` above